src/Quartz/DateBuilder.cs

```diff
@@ -114,7 +102,7 @@ public static DateBuilder NewDate()
     /// <returns></returns>
     public static DateBuilder NewDateInTimeZone(TimeZoneInfo tz)
```

Comment:
maybe needs parameter `TimeProvider? timeProvider = null` and then we default to system?

Reply:
As the DateBuilder constructors are private, I've reduced the constructor to one, made TimeZoneInfo? optional, and as per your comment, in the public method `NewDate` and `NewDateInTimeZone` default to TimeProvider.System when not provided.  Take a look at next PR let me know.